sources/packages/web/src/views/student/StudentFileUploader.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <full-page-container>
```

Comment:
The header is expected in the upcoming PR?

Reply:
yes